content/docs/get-started/aws/modify-program.md

```diff
@@ -86,6 +86,43 @@ bucket = s3.Bucket('my-bucket',
 pulumi.export('bucket_name',  bucket.id)
 ```
 
+```csharp
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Pulumi;
+using Pulumi.Aws.Kms;
+using Pulumi.Aws.S3;
+
+class Program
+{
+    static Task Main() =>
+        Deployment.Run(() =>
+        {
+            // Create a KMS Key for S3 server-side encryption
+            var key = new Kms.Key('my-key');
+
+            // Create an AWS resource (S3 Bucket)
+            var bucket = new S3.Bucket('my-bucket', new S3.BucketArgs
+            {
+                ServerSideEncryptionConfiguration = new S3.BucketServerSideEncryptionConfigurationArgs
+                {
+                    Rules = new S3.BucketServerSideEncryptionConfigurationRuleArgs
+                    {
+                        ApplyServerSideEncryptionByDefault = new S3.BucketServerSideEncryptionConfigurationRuleApplyServerSideEncryptionByDefaultArgs
```

Comment:
Woah...that's a mouthful! Is this the final design, or do we plan on giving friendlier names to these things?

Reply:
Nothing is final design here.  It's all preview.  :)